clickhouse/clickhouse.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+
+# colors
+CRed='\033[1;31m'
+CGray='\033[0;37m'
+CGreen='\033[1;32m'
+CYellow='\033[1;33m'
+NC='\033[0m'
+
+# logging
+info() {
+  echo -e "${CGray}${1}${NC}"
+}
+
+warn() {
+  echo -e "${CYellow}${1}${NC}"
+}
+
+error() {
+  echo -e "${CRed}${1}${NC}"
+}
+
+success() {
+  echo -e "${CGreen}${1}${NC}"
+}
+
+# vars
+# os="${os:-${1}}"
+arch="${arch:-${2}}"
+version="${version:-${3}}"
+
+case $arch in
+amd64) arch_alias="";;
+arm64) arch_alias="aarch64";;
+esac
+
+# https://github.com/ClickHouse/ClickHouse/releases/download/v24.4.1.2088-stable/clickhouse-macos
+# https://github.com/ClickHouse/ClickHouse/releases/download/v24.4.1.2088-stable/clickhouse-macos-aarch64
+
+url="https://github.com/ClickHouse/ClickHouse/releases/download/${version}/clickhouse-macos-${arch_alias}"
```

Comment:
use `os`

Reply:
os in my case is "darwin". That doesn't fit here. Do mean to hard coded it with "macos"?

Reply:
us `os_alias` similar to `arch_alias`. see other implementations

Reply:
done